test/e2e/e2e_test.go

```diff
@@ -54,11 +54,11 @@ func init() {
 func TestMain(m *testing.M) {
 	flag.StringVar(&globalOpts.configurableClusterOptions.AWSCredentialsFile, "e2e.aws-credentials-file", "", "path to AWS credentials")
 	flag.StringVar(&globalOpts.configurableClusterOptions.Region, "e2e.aws-region", "us-east-1", "AWS region for clusters")
-	flag.StringVar(&globalOpts.configurableClusterOptions.Zone, "e2e.aws-zone", "us-east-1a", "AWS zone for clusters")
+	flag.Var(&globalOpts.configurableClusterOptions.Zone, "e2e.aws-zones", "AWS zones for clusters")
```

Comment:
`don't we need to preserve default []string{"us-east-1a"}?`

Reply:
I do that here
https://github.com/openshift/hypershift/pull/964/files#diff-d5d12b6ea07b0a78c9ad1987e1486c565ecf16839f77a93e91b09475cf6c0e99R223